appserver/security/core-ee/src/main/java/com/sun/enterprise/security/ee/SecuritySniffer.java

```diff
@@ -80,11 +81,27 @@
     };
 
     public SecuritySniffer() {
-        super("security", "WEB-INF/web.xml", null);
```

Comment:
This change should not be needed. This is just a shortcut for adding security sniffer for a web archive which contains a WEB-INF/web.xml. If it does not find it, it will invoke the handles method for further looking.